tests/test_transform.py

```diff
@@ -20,6 +21,9 @@
         (0, IntUniformDistribution(0, 3)),
         (1, IntLogUniformDistribution(1, 10)),
         (2, IntUniformDistribution(0, 10, step=2)),
+        (0, IntDistribution(0, 3)),
+        (1, IntDistribution(1, 10)),
```

Comment:
[question] Should  this sample have `log=True`?

Reply:
Thank you very much for pointing out, you are right.